src/components/stepforms/schemas/utils.ts

```diff
@@ -0,0 +1,24 @@
+import _ from 'lodash';
+
+type JsonPropSchema = {
+  [prop: string]: any;
+};
+
+type JsonSchema = {
+  properties: { [prop: string]: JsonPropSchema };
+};
+
+export type StepFormType = {
+  columnNames: string[];
+};
+
+export function addNotInColumnNamesConstraint(
+  model: JsonSchema,
+  stepname: string,
```

Comment:
Actually it is more a filed name that the step name that we use here right ? If so I find it confusing...